src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -1938,6 +1948,7 @@ Real64 ElectPowerLoadCenter::calcLoadCenterThermalLoad()
                                                     plantNotFound);
             if (!plantNotFound) g->plantInfoFound = true;
         }
+        myCoGenSetupFlag_ = false;
```

Comment:
I was expecting to see this as a new member variable, but I guess it's already in there.  In which case this should've been set to false when it was first added.  OK.